core/src/main/java/io/questdb/cutlass/pgwire/PGConnectionContext.java

```diff
@@ -1014,10 +1082,33 @@ private void processClose() throws PeerDisconnectedException, PeerIsSlowToReadEx
         send();
     }
 
-    private void processDescribe() throws PeerDisconnectedException, PeerIsSlowToReadException {
+    private void processDescribe(@Transient ObjList<BindVariableSetter> bindVariableSetters,
+                                 long lo,
+                                 long msgLimit,
+                                 @Transient AssociativeCache<Object> portalFactoryCache,
+                                 @Transient AssociativeCache<Object> statementToBindVariableTypeCache) throws SqlException, BadProtocolException {
+        lo = lo + 1;
+        long hi = getStringLength(lo, msgLimit);
+        checkNotTrue(hi == -1, "bad portal name length [msgType='D']");
+
+        CharSequence statementName = getStatementName(lo, hi);
+        if (statementName != null) {
+            final Object statement = portalFactoryCache.peek(statementName);
```

Comment:
this is another site where `statement` can be null and this situation is unhandled